src/server/logging/AppConsole.ts

```diff
@@ -2,14 +2,23 @@ import * as stackTrace from 'stack-trace';
 
 import { ILogEntry, ILogger, LogMessageSeverity } from '../../definition/accessors';
 import { AppMethod } from '../../definition/metadata';
+import { AppManager } from '../AppManager';
 import { ILoggerStorageEntry } from './ILoggerStorageEntry';
 
 export class AppConsole implements ILogger {
     public static toStorageEntry(appId: string, logger: AppConsole): ILoggerStorageEntry {
+        const entries = logger.getEntries();
+
+        if (logger.isDevelopmentModeEnabled) {
```

Comment:
@lolimay I'm wondering if it wouldn't be better for us to add this part in the logging methods instead of the conversion methods. Maybe there is a scenario where the `toStorageEntry` method isn't reached? What do you think?

Reply:
I didn't put this part into the `addEntry` method because we can't get the app's information in that method 😔:
https://github.com/RocketChat/Rocket.Chat.Apps-engine/blob/8537e5ec27eb0f623ad53b0329ee07987ec9091d/src/server/logging/AppConsole.ts#L74-L95

This may cause chaos if we just print all the logs to the terminal without the app's info. Any idea that we can keep both the apps' info stayed and also all the scenarios been covered?

Reply:
Ah! Ok, that's a very good reason! :D